controllers/workspace/eventhandlers.go

```diff
@@ -106,3 +108,11 @@ func (r *DevWorkspaceReconciler) runningWorkspacesHandler(obj client.Object) []r
 	}
 	return reconciles
 }
+
+func (r *DevWorkspaceReconciler) certificateHandler(obj client.Object) []reconcile.Request {
+	certsPem, ok := obj.(*corev1.ConfigMap).Data["custom-ca-certificates.pem"]
```

Comment:
All configmap keys can contain certificates. 
We shouldn't stick to a single hardcoded key.

Reply:
done